HabitTracker/HabitTracker/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "HabitTracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\AhmadJerjees\\source\\repos\\HabitTracker\\HabitTracker"
```

Comment:
Remember that this hard-coded path will not exist on other peoples machines, you will need to include some info in README on changing this as a prerequisite step

Reply:
oh i never gave it a thought to make a dynamic path that would work on every machine, apologies Mr , i will make sure to fix it.